include/solvers/navier_stokes_base.h

```diff
@@ -338,23 +349,131 @@ class NavierStokesBase : public PhysicsSolver<VectorType>
   check_existance_of_bc(BoundaryConditions::BoundaryType bc);
 
   /**
-   * @brief Turns regions of the mesh where the material_id=1 to a solid block by injecting it into the constraints
-   * This is achieved by imposing $\mathbf{u}=0$ within the cells which have a
-   * material_id=1. In addition, solid cells which are not connected to the
-   * fluid by any means also get a pressure dirichlet boundary condition which
-   * fixes the pressure to 0. This ensures that the linear system is well-posed.
-   * Right now, this routine only supports the usage of 1 solid domain, but
-   * eventually it could be extended to more than one. By default, the fluid
-   * domain is assumed to have a material_id=0 and the rest of the domains have
-   * a material_id>0.
+   * @brief Turn regions of the mesh where the @p material_id>0 into a solid
```

Comment:
This is not really brief. If you could summarize the function is 1-2 sentences and put the complete explanation afterwards it would be better.